uber/configspec.ini

```diff
@@ -49,6 +49,15 @@ use_checkin_barcode = boolean(default=False)
 # registering, False to disallow promo codes.
 badge_promo_codes_enabled = boolean(default=False)
 
+# This controls the window after "prereg_open" during which the "Request
+# Hotel Booking Info" step is including in the preregistration workflow.
+# The duration is expressed as a number of seconds. For example: to keep
+# "Hotel Info" open for a 24 hour window after prereg launches set this
+# value to 86400 (24 hours * 60 minutes * 60 seconds).
```

Comment:
Minutes or hours would also be fine.

Internally I try to record all time durations as seconds (optionally using a float to record fractions of seconds). And then I convert to and from more manageable units when interacting with an end user. So a user may input minutes or hours in a form, but that gets converted to seconds internally.

In this case, however, there is no end-user! Or... there is an end-user, but it's actually a programmer? And it's just as easy for a programmer to get confused as it is for a regular end-user (see the recent pending email report spam incident for an example!).

Whatever we decide, we must clearly indicate the units we are using anywhere this appears in the config.